{"name": "Test Registration with Invalid Data", "status": "broken", "statusDetails": {"message": "AttributeError: 'NoneType' object has no attribute 'is_displayed'", "trace": "self = <paraBank_test.TestParaBank object at 0x000001B6EC89CCD0>\n\n    @allure.title(\"Test Registration with Invalid Data\")\n    def test_register_user_with_invalid_data(self):\n        self.pb.click_register()\n        self.pb.add_customer_details(\n            first_name=\"Jane\",\n            last_name=\"Doe\",\n            address=\"456 Oak St\",\n            city=\"Springfield\",\n            state=\"IL\",\n            zip_code=\"62701\",\n            phone=\"555-5678\",\n            ssn=\"\",\n            username=\"\",\n            password=\"\"\n        )\n        self.pb.click_register_button()\n        with allure.step(\"Verify error message for invalid registration\"):\n            sleep(5)\n>           assert self.pb.get_element(self.pb.Locators.ERROR_USERNAME).is_displayed(), \"Error message not found\"\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE           AttributeError: 'NoneType' object has no attribute 'is_displayed'\n\ntestcases\\paraBank_test.py:64: AttributeError"}, "steps": [{"name": "Clicking on Register link", "status": "passed", "start": 1751094117759, "stop": 1751094119843}, {"name": "Adding customer details: 'Jane' 'Doe'", "status": "passed", "parameters": [{"name": "first_name", "value": "'Jane'"}, {"name": "last_name", "value": "'Doe'"}, {"name": "address", "value": "'456 Oak St'"}, {"name": "city", "value": "'Springfield'"}, {"name": "state", "value": "'IL'"}, {"name": "zip_code", "value": "'62701'"}, {"name": "phone", "value": "'555-5678'"}, {"name": "ssn", "value": "''"}, {"name": "username", "value": "''"}, {"name": "password", "value": "''"}], "start": 1751094119843, "stop": 1751094121284}, {"name": "Clicking on Register button and verifying welcome message", "status": "passed", "start": 1751094121285, "stop": 1751094123573}, {"name": "Verify error message for invalid registration", "status": "broken", "statusDetails": {"message": "AttributeError: 'NoneType' object has no attribute 'is_displayed'\n", "trace": "  File \"C:\\vsCode\\parabank_credentials\\testcases\\paraBank_test.py\", line 64, in test_register_user_with_invalid_data\n    assert self.pb.get_element(self.pb.Locators.ERROR_USERNAME).is_displayed(), \"Error message not found\"\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1751094123573, "stop": 1751094138629}], "attachments": [{"name": "screenshot", "source": "f0c47bbc-4c5d-4973-acfc-f612e23fb549-attachment.png", "type": "image/png"}, {"name": "stdout", "source": "c50da28a-387f-4c2e-a889-c6a94cfaecb6-attachment.txt", "type": "text/plain"}], "start": 1751094117759, "stop": 1751094138631, "uuid": "d10ac5e5-2626-4ca8-908a-a0b8609687f6", "historyId": "c72fc3edbefed83b6e3076b54585fcc5", "testCaseId": "c72fc3edbefed83b6e3076b54585fcc5", "fullName": "testcases.paraBank_test.TestParaBank#test_register_user_with_invalid_data", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "paraBank_test"}, {"name": "subSuite", "value": "TestParaBank"}, {"name": "host", "value": "Hans"}, {"name": "thread", "value": "6008-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.paraBank_test"}]}